api/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>com.flipkart.poseidon</groupId>
         <artifactId>poseidon</artifactId>
-        <version>5.3.0</version>
+        <version>5.4.0</version>
```

Comment:
Let's stay with snapshot... Based on consumption feedback, we can release with/without any further changes.